lib/usd/ui/USDImportDialogCmd.h

```diff
@@ -25,6 +25,8 @@
 MAYAUSD_NS_DEF {
 
 class MAYAUSD_UI_PUBLIC USDImportDialogCmd : public MPxCommand {
+    MStatus applyToProxy(const MString& proxyPath);
```

Comment:
I would prefer if you created a new "private" method section after line 42 with this method. This matches how the rest of this project was coded. Thanks.